content/en/docs/reference/kubernetes-api/common-definitions/list-meta.md

```diff
@@ -44,11 +44,5 @@ ListMeta describes metadata that synthetic resources must have, including lists
 
 - **selfLink** (string)
 
-  selfLink is a URL representing this object. Populated by the system. Read-only.
-  
-  DEPRECATED Kubernetes will stop propagating this field in 1.20 release and the field is planned to be removed in 1.21 release.
-
-
-
-
+  SelfLink is a legacy read-only field that is no longer populated by the system.
```

Comment:
This change needs to happen [upstream](https://github.com/kubernetes/kubernetes/blob/2ccdd4573b6ae500b96e77638752de8f2f329fe4/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L61) first.

Reply:
I opened https://github.com/kubernetes/kubernetes/pull/108163 for it.

Reply:
This is now merged in upstream.